src/api/app/views/webui/user/show.html.haml

```diff
@@ -29,6 +29,9 @@
     %ul
       - @role_titles.each do |title|
         %li= title
+  / beta
+  - if @displayed_user.in_beta?
+    %p Part of the beta
```

Comment:
[nitpick] Participates in public beta program

Reply:
It does sound better